Resfeber/Helpers/UITextField+Extension.swift

```diff
@@ -14,11 +14,12 @@ extension UITextField {
         let screenWidth = UIScreen.main.bounds.width
         let datePicker = UIDatePicker(frame: CGRect(x: 0, y: 0, width: screenWidth, height: 216))//1
         datePicker.datePickerMode = .date
-        // iOS 14 and above
-        if #available(iOS 14, *) {
+        if #available(iOS 13.4, *) {
```

Comment:
If you updated your deployment target to iOS 14, iOS 13.4 will _always_ be available, so you shouldn't need any of this